sensor_msgs/CompressedImage image
string class_name
---
bool success